contract/src/lib.rs

```diff
@@ -40,8 +38,8 @@ construct_uint! {
 /// Interface for the contract itself.
 #[ext_contract(ext_self)]
 pub trait SelfContract {
-    /// A callback to check the result of the nep21-trasnfer
-    fn nep21_transfer_callback(&mut self);
+    /// callback to check the result of the add_liquidity action
+    fn add_liquidity_transfer_callback(&mut self, token:AccountId);
```

Comment:
this is called on nep-21 transfer. Why changing the name?

Reply:
because it's called after add liquidity, we will need this to rolllback, so there's gonna be one callback for each rollbackeable txn

Reply:
but let's leave the rollback for *after* demo day